api_tests/src/wallets/ethereum.ts

```diff
@@ -119,7 +109,32 @@ export class EthereumWallet implements Wallet {
                 this.ethereumDevAccount.address
             );
             const signedTx = await this.ethereumDevAccount.sign(tx);
-            return this.jsonRpcProvider.sendTransaction(signedTx);
+            const transactionResponse = await this.jsonRpcProvider.sendTransaction(
+                signedTx
+            );
+
+            this.logger.debug(
+                "Transaction: ",
+                transactionResponse.hash,
+                " sent, waiting to be confirmed."
+            );
+
+            const transactionReceipt = await this.jsonRpcProvider.waitForTransaction(
+                transactionResponse.hash,
+                1
+            );
+            if (!transactionReceipt.status) {
+                throw new Error(
+                    `Minting ${tx.value} tokens to address ${tx.to} failed`
```

Comment:
This error message is wrong now because we are not just calling this for minting right?

Reply:
That is correct. I rephrased and rebased :)